Gems/EMotionFX/Code/Tests/UI/RagdollEditTests.cpp

```diff
@@ -56,13 +54,16 @@ namespace EMotionFX
 
             EXPECT_CALL(m_jointHelpers, ComputeInitialJointLimitConfiguration(_, _, _, _, _))
                 .WillRepeatedly(
-                    []([[maybe_unused]] const AZ::TypeId& jointLimitTypeId, [[maybe_unused]] const AZ::Quaternion& parentWorldRotation,
-                       [[maybe_unused]] const AZ::Quaternion& childWorldRotation, [[maybe_unused]] const AZ::Vector3& axis,
+                    []([[maybe_unused]] const AZ::TypeId& jointLimitTypeId,
+                       [[maybe_unused]] const AZ::Quaternion& parentWorldRotation,
+                       [[maybe_unused]] const AZ::Quaternion& childWorldRotation,
+                       [[maybe_unused]] const AZ::Vector3& axis,
                        [[maybe_unused]] const AZStd::vector<AZ::Quaternion>& exampleLocalRotations)
                     {
                         return AZStd::make_unique<D6JointLimitConfiguration>();
                     });
 
+            UIFixture::SetUp();
```

Comment:
question: Is it wise to move this after? Or is this to avoid the spam for the expect calls? 🤔 It might be worth adding a comment to explain why this isn't the first thing as usually it's good practice to call these things in the right order (so all the base stuff definitely happens first)

Reply:
it's exactly for that reason. i'll add an comment